### PR TITLE
RATIS-1940. Add pluggable metrics description into project website

### PR DESCRIPTION
**Pull request by SzyWilliam**
https://issues.apache.org/jira/browse/RATIS-1940.

See the result:
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,6 +152,17 @@ <h4>Pluggable raft log</h4>
                   Application can define the policies of how to write data and where it should be written easily.
               </p>
           </div>
+
+          <div class="col-md-6 col-sm-6 feature-item">
+              <span class="glyphicon glyphicon-dashboard" aria-hidden="true"></span>
+              <h4>Pluggable metrics</h4>
+              <p>Ratis provides a pluggable observation layer. By default, it uses dropwizard4. Users can provide their
```
Comment (szetszwo) [inline index.html:159]: @SzyWilliam , thanks for working on this!  Let's also mention `ratis-metrics-dropwizard3`.
```
By default (ratis-metrics-default), it uses the shaded Dropwizard 4 provided by ratis-thirdparty.
Another implementation can be found in ratis-metrics-dropwizard3.
```

Comment (SzyWilliam) [reply index.html:159]: Sure! Let me change the description.

**Review by szetszwo — COMMENTED** [review]
+1 the change looks good.

**Review by szetszwo — APPROVED** [review]

**Comment by SzyWilliam** [comment]
@szetszwo Thanks for reviewing this PR!

**Comment by szetszwo** [comment]
Please also push the source.  :)

BTW, we should have pull requests for merging to `asf-site-source`.  Once the source has been merged, we may generate `asf-site` and push it without another PR.

**Comment by SzyWilliam** [comment]
> BTW, we should have pull requests for merging to asf-site-source. Once the source has been merged, we may generate asf-site and push it without another PR.

Thanks for telling me the best-practice of asf-site! I was wondering before why we need both asf-site-source and asf-site and now I learned why. I'll update the asf-site-source and generate the website next time.

**Pull request merged**